### PR TITLE
Updating icons for Sofort, Bancontact and iDeal

### PR DESCRIPTION
**Pull request by mseal-shopify**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.
- [x] I recognize that if my icon is not approved by the Shopify Partners team it may not receive review nor merger.

I have obtained the icons from:
https://www.sofort.com/integration-guidelines/
https://www.bancontact.com/nl/promotiemateriaal/richtlijnen-voor-het-logo
https://www.ideal.nl/en/businesses/logos/.

### CODE REVIEW

**Comment by mseal-shopify** [comment]
I was unable to set the width and height to 38 and 24 respectively as the icons were not resizing.

**Comment by mseal-shopify** [comment]
Update the height and width of the files.

File: app/assets/images/payment_icons/sofort.svg [hunk 1]
```diff
@@ -1 +1 @@
...
... 1.026 1.19.008.796-.521 1.515-1.275 1.515zM21.18 9.28c-1.58 0-2.772 1.353-2.772 2.965 0 1.458.985 2.475 2.326 2.475 1.58 0 2.773-1.391 2.773-3.003.008-1.468-.977-2.437-2.327-2.437zm-.347 4.068c-.604 0-1.026-.461-1.026-1.171 0-.787.521-1.534 1.275-1.534.603 0 1.026.48 1.026 1.19 0 .796-.521 1.515-1.276 1.515h.001zm7.325-2.35c0-.98-.654-1.603-1.846-1.603H24.64l-.736 5.21h1.407l.232-1.641h.066l.744 1.641h1.614l-1.034-1.824c.77-.307 1.224-.968 1.224-1.783h.001zm-2.251.864H25.7l.157-1.095h.19c.439 0 .67.154.67.49 0 .394-.33.605-.81.605zm-18.63-.385c-.538-.317-.655-.394-.655-.567 0-.21.215-.316.505-.316.34 0 .853.048 1.358.604A3.63 3.63 0 019.08 9.99c-.71-.47-1.374-.71-1.977-.71-1.2 0-1.895.748-1.895 1.65 0 .778.504 1.18 1.1 1.536.538.316.67.422.67.614 0 .211-.223.326-.521.326-.513 0-1.11-.374-1.465-.758l-.207 1.449a2.52 2.52 0 001.697.624c1.241 0 1.912-.73 1.912-1.661 0-.768-.43-1.18-1.117-1.583zm10.428.201H16.05l.132-.912h1.77a3.538 3.538 0 011.027-1.37h-2.897c-.645 0-1.191.508-1.283 1.199l-.562 4.02h1.407l.232-1.642h1.672l.124-.891c.008-.144.017-.27.034-.404h-.001zM28.19 9.396c.38.336.612.806.662 1.371h1.175l-.537 3.838h1.406l.538-3.838h1.308l.19-1.372h-4.742z" fill="#EDEDED"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/sofort.svg:1]: - The icon missing the border. [video demonstration](https://screenshot.click/07-59-m2etv-dlw1l.mp4)
- Please remove the grey background `<path fill="#F5F5F5" d="M0 0h38v24H0z"/>`

The [guidelines](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md#template) provided an example that you can take from.

If I try to add it myself, the icon's branding background color goes through the border. You might want to resize down the icon further.

File: app/assets/images/payment_icons/ideal.svg [hunk 2]
```diff
@@ -1 +1 @@
...
....39-16.163 10.39zM4.705 2.206c-.881 0-1.589.507-1.589 1.139v17.32c0 .63.708 1.138 1.589 1.138h14.823c9.906 0 15.368-3.487 15.368-9.82 0-8.505-9.633-9.777-15.368-9.777H4.705z" fill="#000"/><path d="M14.636 9.945c.323 0 .621.036.907.107.285.071.521.187.732.329.199.151.36.338.484.551.112.223.174.48.174.783 0 .267-.05.507-.136.73-.1.222-.236.418-.423.578-.186.16-.422.285-.707.383-.286.088-.62.142-1.006.142H12.49V9.936h2.147v.01zm-.074 2.944c.161 0 .31-.017.472-.053.149-.035.285-.098.397-.187a1.05 1.05 0 00.285-.338c.075-.142.112-.302.112-.507a1.51 1.51 0 00-.074-.489.894.894 0 00-.249-.374 1.161 1.161 0 00-.447-.24 2.487 2.487 0 00-.682-.08h-.795v2.277h.98v-.008zm6.853-2.944v.667h-2.657v.774h2.445V12h-2.445v.88h2.719v.668h-3.824V9.936h3.762v.01zm3.798 0l1.887 3.612h-1.154l-.385-.8h-1.887l-.397.8h-1.118l1.9-3.612h1.154zm.062 2.215l-.633-1.325h-.012l-.658 1.325h1.303zM28.9 9.945v2.944h2.458v.668h-3.563V9.945H28.9z" fill="#fff"/><path d="M7.262 13.414c1.282 0 2.322-.744 2.322-1.663s-1.04-1.663-2.322-1.663-2.321.744-2.321 1.663 1.04 1.663 2.321 1.663zM9.013 20.122c-1.95 0-3.513-1.13-3.513-2.518v-1.966c0-.694.782-1.263 1.762-1.263.969 0 1.763.56 1.763 1.263v4.484h-.012z" fill="#000"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/ideal.svg:1]: See my previous comment. We should also apply the same changes here.

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
We'll need to make a few changes on those 2 icons:
- remove grey background color
- missing border

**Comment by mseal-shopify** [comment]
Hi @tauthomas01 , I've updated the icons based on your comments. Please let me know if they seem alright now.

File: app/assets/images/payment_icons/sofort.svg (hunk 1 above)
Comment (mseal-shopify) [reply app/assets/images/payment_icons/sofort.svg]: Updated.

File: app/assets/images/payment_icons/ideal.svg (hunk 2 above)
Comment (mseal-shopify) [reply app/assets/images/payment_icons/ideal.svg]: uopdated.

**Comment by larouxn** [comment]
Looks like at least one of the icons is missing a `title` element. ([CI logs](https://github.com/activemerchant/payment_icons/runs/6804543070?check_suite_focus=true))

```
Failure:
PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [/home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:68]:
{:message=>"The 'ideal' SVG file should have a single <title> tag"}.
Expected: 1
  Actual: 0
```

**Comment by mseal-shopify** [comment]
@larouxn and @tauthomas01 Thanks for the help. Could I ask for one favour? Could you help me with the iDeal icon? I tried to resize it, but it gets squished and starts to look weird. I have reverted the resize of the icon. I even added the base template and optimized it, but neither does the border show up, nor is the size correct.

File: app/assets/images/payment_icons/ideal.svg [hunk 3]
```diff
@@ -1 +1 @@
...
....256-4.538 11.255-12.76 11.255ZM3.714 2.39A1.24 1.24 0 0 0 2.46 3.623v18.763a1.24 1.24 0 0 0 1.254 1.234h11.702c7.822 0 12.134-3.778 12.134-10.64 0-9.212-7.605-10.59-12.134-10.59H3.714Z"/><path fill="#fff" d="M11.555 10.774c.255 0 .49.039.716.116.225.077.411.202.578.356.157.164.284.366.382.598.088.24.137.52.137.848 0 .289-.039.549-.108.79a1.832 1.832 0 0 1-.333.626 1.616 1.616 0 0 1-.559.415c-.225.096-.49.154-.793.154H9.86v-3.913h1.695v.01Zm-.059 3.19c.128 0 .245-.02.373-.058a.732.732 0 0 0 .313-.203 1.12 1.12 0 0 0 .226-.366c.059-.154.088-.327.088-.549 0-.193-.02-.376-.059-.53a1.046 1.046 0 0 0-.196-.405.883.883 0 0 0-.353-.26 1.466 1.466 0 0 0-.539-.087h-.627v2.467h.774v-.01ZM16.906 10.774v.723H14.81v.838h1.93V13h-1.93v.954h2.146v.723h-3.018v-3.913h2.97v.01ZM19.905 10.774l1.49 3.912h-.912l-.303-.867h-1.49l-.314.867h-.882l1.5-3.912h.911Zm.05 2.4-.5-1.436h-.01l-.52 1.436h1.03ZM22.816 10.774v3.19h1.94v.722h-2.812v-3.912h.872Z"/><path d="M5.733 14.532c1.013 0 1.833-.807 1.833-1.802s-.82-1.802-1.833-1.802c-1.012 0-1.832.807-1.832 1.802s.82 1.802 1.832 1.802ZM7.115 21.798c-1.538 0-2.773-1.224-2.773-2.727v-2.13a1.38 1.38 0 0 1 1.391-1.368 1.38 1.38 0 0 1 1.392 1.368v4.857h-.01Z"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/ideal.svg:1]:
```suggestion
<svg viewBox="0 0 38 24" xmlns="http://www.w3.org/2000/svg" width="38" height="24" role="img" aria-labelledby="pi-ideal"><title id="pi-ideal">iDEAL</title><path opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3Z"/><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32Z" fill="#fff"/><path d="M14 6.912V19h5.648C24.776 19 27 16.302 27 12.486 27 8.834 24.776 6 19.648 6h-4.67c-.543 0-.978.414-.978.912Z" fill="#C06"/><path d="M19.312 21h-8.884C9.64 21 9 20.373 9 19.6V5.4c0-.773.64-1.4 1.428-1.4h8.884C27.742 4 29 9.317 29 12.482 29 17.974 25.555 21 19.313 21h-.001ZM10.428 4.467a.944.944 0 0 0-.878.573.936.936 0 0 0-.074.36v14.2a.936.936 0 0 0 .59.866c.115.046.238.07.362.068h8.884c5.938 0 9.212-2.86 9.212-8.052 0-6.972-5.774-8.015-9.212-8.015h-8.884Z"/><path d="M16.252 11.008c.188 0 .361.03.528.088.167.06.304.155.427.273.116.125.21.28.282.457.065.184.101.398.101.649 0 .22-.028.42-.08.604a1.417 1.417 0 0 1-.245.479 1.197 1.197 0 0 1-.413.317 1.437 1.437 0 0 1-.586.118H15V11h1.252v.008Zm-.044 2.44c.095 0 .181-.016.276-.045a.539.539 0 0 0 .23-.155.863.863 0 0 0 .168-.28c.043-.118.065-.25.065-.42 0-.147-.015-.287-.044-.405a.814.814 0 0 0-.145-.31.656.656 0 0 0-.26-.199 1.047 1.047 0 0 0-.398-.066h-.464v1.887h.572v-.008Zm3.995-2.44v.553h-1.548v.64h1.426v.51h-1.426v.73h1.585v.552h-2.229V11h2.194v.008h-.002Zm2.215 0 1.1 2.992h-.673l-.224-.663h-1.1l-.232.663h-.652l1.108-2.992h.673Zm.037 1.835-.37-1.098h-.007l-.384 1.098h.76Zm2.112-1.835v2.44H26V14h-2.076v-2.992h.643Z" fill="#fff"/><path d="M11.5 13.652c.829 0 1.5-.593 1.5-1.326 0-.732-.671-1.326-1.5-1.326s-1.5.594-1.5 1.326c0 .732.671 1.326 1.5 1.326ZM12.63 19c-1.258 0-2.269-.9-2.269-2.007v-1.568a.969.969 0 0 1 .337-.715c.214-.189.502-.294.802-.291a1.24 1.24 0 0 1 .433.073c.137.05.262.124.368.218.106.093.19.205.248.327a.93.93 0 0 1 .09.388V19h-.008Z"/></svg>
```

File: app/assets/images/payment_icons/sofort.svg [hunk 4]
```diff
@@ -1 +1 @@
...
...8 1.026 1.19.008.796-.521 1.515-1.275 1.515zM21.18 9.28c-1.58 0-2.772 1.353-2.772 2.965 0 1.458.985 2.475 2.326 2.475 1.58 0 2.773-1.391 2.773-3.003.008-1.468-.977-2.437-2.327-2.437zm-.347 4.068c-.604 0-1.026-.461-1.026-1.171 0-.787.521-1.534 1.275-1.534.603 0 1.026.48 1.026 1.19 0 .796-.521 1.515-1.276 1.515h.001zm7.325-2.35c0-.98-.654-1.603-1.846-1.603H24.64l-.736 5.21h1.407l.232-1.641h.066l.744 1.641h1.614l-1.034-1.824c.77-.307 1.224-.968 1.224-1.783h.001zm-2.251.864H25.7l.157-1.095h.19c.439 0 .67.154.67.49 0 .394-.33.605-.81.605zm-18.63-.385c-.538-.317-.655-.394-.655-.567 0-.21.215-.316.505-.316.34 0 .853.048 1.358.604A3.63 3.63 0 019.08 9.99c-.71-.47-1.374-.71-1.977-.71-1.2 0-1.895.748-1.895 1.65 0 .778.504 1.18 1.1 1.536.538.316.67.422.67.614 0 .211-.223.326-.521.326-.513 0-1.11-.374-1.465-.758l-.207 1.449a2.52 2.52 0 001.697.624c1.241 0 1.912-.73 1.912-1.661 0-.768-.43-1.18-1.117-1.583zm10.428.201H16.05l.132-.912h1.77a3.538 3.538 0 011.027-1.37h-2.897c-.645 0-1.191.508-1.283 1.199l-.562 4.02h1.407l.232-1.642h1.672l.124-.891c.008-.144.017-.27.034-.404h-.001zM28.19 9.396c.38.336.612.806.662 1.371h1.175l-.537 3.838h1.406l.538-3.838h1.308l.19-1.372H28.19z" fill="#EDEDED"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/sofort.svg:1]:
```suggestion
<svg width="38" height="24" viewBox="0 0 38 24" fill="none" xmlns="http://www.w3.org/2000/svg" role="img" aria-labelledby="pi-sofort"><title id="pi-sofort">SOFORT</title><path opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3Z" fill="#000"/><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32Z" fill="#fff"/><path d="M33.555 20H3L5.322 5H36l-2.445 15Z" fill="#393A41"/><path d="M13.116 10c-1.403 0-2.47 1.243-2.47 2.725 0 1.34.875 2.274 2.066 2.274 1.403 0 2.462-1.278 2.462-2.76.008-1.349-.867-2.239-2.058-2.239Zm-.315 3.738c-.538 0-.912-.423-.912-1.076 0-.723.463-1.41 1.132-1.41.536 0 .911.441.911 1.094.008.731-.462 1.392-1.132 1.392ZM21.56 10c-1.402 0-2.461 1.243-2.461 2.725 0 1.34.875 2.274 2.066 2.274 1.403 0 2.463-1.278 2.463-2.76C23.636 10.89 22.76 10 21.562 10Zm-.307 3.738c-.537 0-.912-.423-.912-1.076 0-.723.463-1.41 1.133-1.41.535 0 .911.441.911 1.094 0 .731-.463 1.392-1.133 1.392Zm6.506-2.16c0-.9-.581-1.472-1.64-1.472h-1.485l-.654 4.787h1.25l.206-1.508h.059l.66 1.508h1.434l-.918-1.676c.684-.282 1.087-.89 1.087-1.638Zm-2 .795h-.183l.139-1.007h.169c.39 0 .595.142.595.45 0 .363-.293.557-.72.557Zm-16.547-.354c-.477-.292-.581-.362-.581-.521 0-.193.19-.29.448-.29.302 0 .758.043 1.206.554.108-.4.287-.777.529-1.11-.63-.431-1.22-.652-1.756-.652-1.066 0-1.683.687-1.683 1.516 0 .715.447 1.085.977 1.412.478.29.595.387.595.564 0 .194-.198.3-.463.3-.456 0-.986-.344-1.301-.697L7 14.426c.417.378.954.582 1.507.574 1.103 0 1.699-.671 1.699-1.527 0-.705-.382-1.084-.993-1.454Zm9.263.184h-1.47l.117-.837h1.572c.2-.491.513-.923.912-1.26h-2.573c-.573 0-1.058.467-1.14 1.102l-.499 3.694h1.25l.206-1.508h1.485l.11-.82c.007-.132.015-.247.03-.37Zm9.312-2.096c.338.308.544.74.588 1.26h1.044l-.477 3.526h1.249l.477-3.527h1.162l.169-1.26h-4.212Z" fill="#EDEDED"/></svg>
```

**Review by tauthomas01 — COMMENTED** [review]
I applied the fix on the icon. Both icons should look good now.

**Comment by mseal-shopify** [comment]
> I applied the fix on the icon. Both icons should look good now.

Thank you sooooo much Thomas!

**Review by larouxn — APPROVED** [review]
Looks good to me 🎩 
Screen Shot 2022-06-10 at 17 15 54

**Comment by larouxn** [comment]
@tauthomas01, icons are looking good to me. How about you? Hope we can get these merged for the next release.
- https://github.com/activemerchant/payment_icons/pull/637

**Review by tauthomas01 — APPROVED** [review]

**Comment by larouxn** [comment]
👋 Hello, your icon(s) have been released in the latest version of this library. The rollout process will begin within a few days. We wanted to have it/them live this week but looking more likely to be live early to mid next week. Will report back here when live. Thank you for your patience. 🙇‍♂️

**Comment by larouxn** [comment]
👋 Hello, actually your icon(s) should be live everywhere now. 🚀

**Pull request merged**